client/helpers/confirm-protocols-affected.js

```diff
@@ -14,18 +14,25 @@ function getProtocolParts(protocols) {
   }).join('');
 }
 
-const confirmRemove = (protocolRef, singularName, key) => (project, item) => {
+const confirmProtocolsAffected = (changeType, protocolRef, singularName, key, label) => (project, item) => {
```

Comment:
I was going to say can we have an object for the first arg? as it's impossible to tell in schema which args are expected in which order